pkg/nodeconfig/nodeconfig.go

```diff
@@ -356,6 +356,38 @@ func (nc *nodeConfig) configureCNI() error {
 	return nil
 }
 
+// ErrWriter is a wrapper to enable error-level logging inside kubectl drainer implementation
```

Comment:
Such structs should not be added to the middle of a file but rather at the end. This will make it easier to find and maintain the file as it grows.

Reply:
I am fine with that too.

Reply:
Refactored in https://github.com/openshift/windows-machine-config-operator/pull/676/commits/febc428b765ed99b8928424f79d04079bb75f331